webapp-django/crashstats/settings/base.py

```diff
@@ -383,20 +353,245 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
+#         'Bugs': {
+#             'api_whitelist': json.dumps({
+#                 'hits': ('id','signature',)
+#             })
+#         }
+#     }
+# })
+
+# ------------------------------------------------
+# Below are settings that can be overridden using
+# environment variables.
+
+CACHE_MIDDLEWARE = config('CACHE_MIDDLEWARE', False, cast=bool)
+# creates "./models-cache" dir
+# only applicable if CACHE_MIDDLEWARE is True
+CACHE_MIDDLEWARE_FILES = config('CACHE_MIDDLEWARE_FILES', True, cast=bool)
+
+# Socorro middleware instance to use
+MWARE_BASE_URL = config('MWARE_BASE_URL', 'http://localhost:5200')
+MWARE_USERNAME = config('MWARE_USERNAME', None)
+MWARE_PASSWORD = config('MWARE_PASSWORD', None)
+# HTTP/1.1 Host header to pass - in case this is a VHost
+MWARE_HTTP_HOST = config('MWARE_HTTP_HOST', None)
+
+DEFAULT_PRODUCT = config('DEFAULT_PRODUCT', 'WaterWolf')
+
+# can be changed from null to log to test something locally
+# or if using the debug toolbar, you might give toolbar a try
+STATSD_CLIENT = config('STATSD_CLIENT', 'django_statsd.clients.null')
+
+# for local development these don't matter
+STATSD_HOST = config('STATSD_HOST', 'localhost')
+STATSD_PORT = config('STATSD_PORT', 8125, cast=int)
+STATSD_PREFIX = config('STATSD_PREFIX', None)
+
+# Enable this to be able to run tests
+# NB: Disable this caching mechanism in production environment as
+# it will break work of anonymous CSRF if there is more than one
+# web server thread.
+# Comment out to use memcache from settings/base.py
+CACHES = {
+    'default': {
+        # use django.core.cache.backends.memcached.MemcachedCache for prod
+        'BACKEND': config(
+            'CACHE_BACKEND',
+            'django.core.cache.backends.locmem.LocMemCache',
+        ),
+        'LOCATION': config(
+            'CACHE_LOCATiON',
+            'crashstats',
+        ),
+        'TIMEOUT': config('CACHE_TIMEOUT', 500),
+        'KEY_PREFIX': config('CACHE_KEY_PREFIX', 'crashstats'),
+    }
+}
+
+TIME_ZONE = config('TIME_ZONE', 'UTC')
+
+# FIXME should support URLs here!
+DATABASES = {
+    'default': {
+        # use django.db.backends.postgresql_psycopg for production
+        'ENGINE': config('DATABASE_ENGINE', 'django.db.backends.sqlite3'),
+        'NAME': config('DATABASE_NAME', 'sqlite.crashstats.db'),
+        'USER': config('DATABASE_USER', ''),
+        'PASSWORD': config('DATABASE_PASSWORD', ''),
+        'HOST': config('DATABASE_HOST', ''),
+        'PORT': config('DATABASE_PORT', ''),
+        'OPTIONS': {
+        },
+        'TEST_CHARSET': 'utf8',
+        'TEST_COLLATION': 'utf8_general_ci',
+    },
+    # 'slave': {
+    #     ...
+    # },
+}
+
+
+# Uncomment this and set to all slave DBs in use on the site.
+SLAVE_DATABASES = config('SLAVE_DATABASES', '', cast=Csv())
+
+# Debugging displays nice error messages, but leaks memory. Set this to False
+# on all server instances and True only for development.
+DEBUG = TEMPLATE_DEBUG = config('DEBUG', True, cast=bool)
```

Comment:
I thought we had agreed we'd make the default be production-geared? The whole security argument.